apis/nucleus/src/components/ActionsToolbar.jsx

```diff
@@ -176,6 +176,10 @@ function ActionsToolbar({
     action: () => setShowMoreItems(!showMoreItems),
   };
 
+  const handleMouseDown = (e) => {
+    e.stopPropagation(); // Prevent clickOutSide handler confirming selection
+  };
+
```

Comment:
This one is a bit strange? This should affect all charts as well and seems like a rather big deal?

Reply:
I also feel a little bit sceptic about this change. I did test it running with the sn-scatter-plot on a mashup and clicking outside the selection toolbar still confirms the selections (which it should do). So this confuses me a little bit, could you explain some more around this @DanielS-Qlik?

Reply:
Should add `'.njs-action-toolbar-popover'` to the included elements list here: https://github.com/qlik-oss/nebula.js/blob/master/apis/nucleus/src/components/listbox/ListBoxPortal.jsx#L76
In that call to useObjectSelections.

Reply:
I did try adding `'.njs-action-toolbar-popover'` at first, but the `event.target` here: https://github.com/DanielS-Qlik/nebula.js/blob/a420ff2253fc38557b5ea46e02a7281c27b6bfa9/apis/nucleus/src/hooks/useObjectSelections.js#L37 was an svg, so the `contains()` would return false. Now it works properly though, and I can not reproduce the previous problem. So I changed as you suggested

Reply:
It looks like I accidentally removed `pointerEvents: 'auto'` here: https://github.com/DanielS-Qlik/nebula.js/blob/a420ff2253fc38557b5ea46e02a7281c27b6bfa9/apis/nucleus/src/components/ActionsToolbar.jsx#L244 when styling, that's why adding `'.njs-action-toolbar-popover'` in the call to useObjectSelection didn't work at first 😬 .